testing/test-helper.js

```diff
@@ -18,13 +18,12 @@ import {xhrServiceForTesting} from '../src/service/xhr-impl';
 import {getService, getServiceForDoc} from '../src/service';
 
 export function stubService(sandbox, win, serviceId, method) {
-  const stub = sandbox.stub();
-  getService(win, serviceId, () => {
-    const service = {};
-    service[method] = stub;
-    return service;
+  const service = getService(win, serviceId, () => {
+    return {
+      [method]: () => {},
```

Comment:
That would create the method `#method` on the object, which is not the value of `method` variable.

```js
let method = 'test';
let obj = {
  method: () => {}
};
obj.test(); // Error
obj.method(); // Fine

obj = {
  [method]: () => {} // This interpolates the value of `method` as the method name
};

obj.test(); // Fine
obj.method(); // Error